webapp/src/main/java/com/box/l10n/mojito/security/HeaderSecurityConfig.java

```diff
@@ -71,4 +86,12 @@ public String getServiceDelimiter() {
   public void setServiceDelimiter(String serviceDelimiter) {
     this.serviceDelimiter = serviceDelimiter;
   }
+
+  public Pattern getForbiddenServicePattern() {
+    return forbiddenServicePattern;
+  }
+
+  public void setForbiddenServicePattern(Pattern forbiddenServicePattern) {
```

Comment:
Do we need this setter? We set the pattern on init, is there ever a use case where we would allow that to change afterwards?